<launch>
 <!-- This assumes roscore is running on this machine or the robot -->
	<node pkg="sdu_rsd_waypoint" type="sdu_rsd_waypoint_node" name="waypoint" output="screen">
		<param name="waypoint_sub" value="/waypoint_goto" />
		<param name="cmd_vel_pub" value="/fmCommand/cmd_vel" />
		<param name="waypointreached_pub" value="/waypoint_reached" />
		<param name="odometry_sub" value="/fmKnowledge/pose" />
		<!--<param name="odometry_sub" value="/slam_pose" />-->
		<param name="laserscan_sub" value="/LaserScanner/scan" />
		<param name="laserscan_frame" value="/laser" />

		<param name="max_turn_output" value="10" /> <!-- Angle difference to turn on spot -->

		<param name="waypoint_reached_threshold" value="0.05" /> <!-- Threshold of when the waypoint is reached. -->
		<param name="waypoint_reached_threshold_angle" value="3" /> <!-- Threshold of when the waypoint is reached (Angle). -->

		<param name="max_distance_obstacle" value="0.50" />
		<param name="speed_obstacle" value="0.5" />
		<param name="min_closepoints" value="15" />
		<param name="clearance_distance" value="0.2" />

		<param name="kp_obstacle" value="0.05" />
		<param name="kd_obstacle" value="0.002" />

		<param name="kp_angle" value=".01" />
		<param name="ki_angle" value="0.0" />
		<param name="kd_angle" value="0.00" />

		<param name="kp_distance" value="0.5" />
		<param name="ki_distance" value="0.0" />
		<param name="kd_distance" value="0.01" />
	</node>
</launch>
